src/Events/Updates/Provider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace TEC\Events\Updates;
+
+use tad_DI52_ServiceProvider as Service_Provider;
+use TEC\Events\Custom_Tables\V1\Provider_Contract;
+use Tribe__Main;
+
+class Provider extends Service_Provider implements Provider_Contract {

```

Comment:
Doc-block missing.

Reply:
This is a provider that hooks on one action another class that has one method.
Make this a controller and simplify the code.
It can be broken down if, and when, required. 
Redirect tests on the controller, extend the controller test case (e.g. [this](https://github.com/the-events-calendar/event-tickets/blob/b681bdd577b69dd617a1d05d97c150f0a4dd011b/tests/ft_integration/TEC/Tickets/Flexible_Tickets/BaseTest.php#L18)).